contracts/multisend/MultisendImplementation.sol

```diff
@@ -59,6 +60,22 @@ contract MultisendImplementation is
         }
     }
 
+    function multisendWithDifferentAmounts(uint256[] amounts, address payable[] memory recipients) public payable nonReentrant {
```

Comment:
![image](https://github.com/user-attachments/assets/089a4308-322d-4c18-bc89-9db9725ec58a)

Is this true?